match2/wikis/valorant/big_match.lua

```diff
@@ -30,10 +35,8 @@ function BigMatch.run(frame)
 	match['matchid'] = identifiers[2]
 	Match.store(match)
 
-	local tournament = {
-		name = args.tournament or '',
-		link = args.tournamentlink or ''
-	}
+	mw.log('BigMatch')
+	mw.logObject(match)
```

Comment:
Remove

Reply:
> LGTM, hopefully we can avoid PRs like this from now on

i hope so too

we should be on the same lvl as the live versions now, so if we PR changes it should be good now i think^^

Reply:
Cool :)